tests/e2e/kyma_cli_e2e_test.go

```diff
@@ -0,0 +1,119 @@
+package e2e_test
+
+import (
+	"github.com/kyma-project/cli/internal/cli"
+	. "github.com/kyma-project/cli/tests/e2e"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Kyma Deployment, Enabling and Disabling Module", Ordered, func() {
```

Comment:
Describe("Describe Kyma Module state control",

Reply:
I feel like this Describe doesn't clearly identify the purpose of the test. I mean we want to test the deployment, enable and disable commands not just the kyma module state

Reply:
Aren't the **enable** and **disable** commands change the state of the deployed module?

Reply:
They add/ remove the module to/from Kyma